### PR TITLE
PT-1345 try fix vulnerabilities

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Yourloops data visualization for diabetes device data
 This node package is a dependency of Blip and was forked from tidepool/sundial.
 
+## Unreleased
```
Comment (ydrisR) [inline CHANGELOG.md]: Don't you want to release this version so it can be used in other projects?

File: test/test_datetimeWrapper.js [hunk 2]
```diff
@@ -603,7 +603,7 @@ describe('sundial', function() {
         expect(tzs.bigFour.length).to.equal(4);
         expect(tzs.unitedStates.length).to.equal(9);
         expect(tzs.hoisted.length).to.equal(98);
-        expect(tzs.theRest.length).to.equal(448);
+        expect(tzs.theRest.length).to.equal(449);
```
Comment (ydrisR) [inline test/test_datetimeWrapper.js:608]: why? Maybe a little comment in the commit would help ;)

**Review by ydrisR — CHANGES_REQUESTED** [review]

File: test/test_datetimeWrapper.js (hunk 2 above)
Comment (yvon-dblg) [reply test/test_datetimeWrapper.js:608]: It was the same the last time we upgrade the moment-timezone package.
The number of timezone in the https://www.iana.org/time-zones have changed...

File: CHANGELOG.md (hunk 1 above)
Comment (yvon-dblg) [reply CHANGELOG.md]: I'm not sure to understand what you mean

**Review by ydrisR — APPROVED** [review]

**Pull request merged**